EndaYohannesQollaGable/EYQG021.xml

```diff
@@ -0,0 +1,125 @@
+                         <textLang mainLang="gez"/>
+                          
+                       </msItem>
+                                                                     
+                        </msContents>
+                   <physDesc>
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>                          
+                                           <material key="parchment"/>
+                                       </support>
+                                       <extent>
+                                           <measure unit="leaf"></measure>                                            
+                                           <dimensions type="outer">
+                                               <height unit="mm">220</height>
+                                               <width unit="mm" cert="low">170</width>
+                                               <depth unit="mm">50</depth>
+                                           </dimensions>
+                                       </extent>
+                            <foliation>Unfoliated.</foliation>
+                            <condition key="intact"/>
+                         </supportDesc>                         
+                     </objectDesc>
+                       
+                       <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                               <seg type="script">Mid-19th/mid-20th century.</seg>
+                               <seg type="ink">Black, red.</seg>
+                                                          
+                               <desc>Left-sloping hand.</desc>
+                           </handNote>
+                       </handDesc> 
+                     
+                       
+                         <bindingDesc>
+                             <binding contemporary="true">
+                                 <decoNote xml:id="b1">Two wooden boards; blind-tooled leather cover; textile inlays.</decoNote>
+                                 <decoNote xml:id="b2" type="bindingMaterial">                                    
+                                     <material key="wood"/>
+                                     <material key="leather"/>                          
+                                     <material key="textile"/>      
+                                 </decoNote>
+                                 
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>
+                     
+                  
+                    
+                      <history>
+                                                <origin>
+                                  <origDate notBefore="1850" notAfter="1949">Mid-19th/mid-20th century.</origDate>
+                                  Poorly written.</origin>
```

Comment:
does `poorly written` belong to the `origin`?

Reply:
Changed.